buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All

```diff
@@ -56,22 +57,24 @@ timeout(time: 10, unit: 'HOURS') {
         // update build description
         currentBuild.description = "OpenJ9: ${OPENJ9_SHA}<br/>OMR: ${OMR_SHA}"
 
-        SDK_VERSIONS.each { SDK_VERSION ->
-            def OPENJDK_REPO = OPENJDK_REPOS["${SDK_VERSION}"]
-            def OPENJDK_BRANCH = OPENJDK_BRANCHES["${SDK_VERSION}"]
-            def OPENJDK_SHA = buildFile.get_sha(OPENJDK_REPO, OPENJDK_BRANCH)
+        echo "Building OpenJ9 extensions for OpenJDK"
+        echo "OPENJ9_SHA:${OPENJ9_SHA}"
+        echo "OMR_SHA:${OMR_SHA}"
+
+        def OPENJDK_SHAS = [:]
+        OPENJDK_REPOS.each { RELEASE, REPO ->
+            OPENJDK_SHAS["${RELEASE}"] = buildFile.get_sha(OPENJDK_REPOS["${RELEASE}"], OPENJDK_BRANCHES["${RELEASE}"])
 
-            // append OpenJDK SHA to the build description
-            currentBuild.description += "<br/>OpenJDK${SDK_VERSION}: ${OPENJDK_SHA}"
+            currentBuild.description += "<br/>OpenJDK${RELEASE}: " + OPENJDK_SHAS[RELEASE]
+            echo "OPENJDK${RELEASE}_SHA:" + OPENJDK_SHAS[RELEASE]
```

Comment:
These two lines use `RELEASE` without quotes or dollar or braces. Can we either change the other lines or change these lines so they are consistent.

Reply:
wrapped in quotes

Reply:
@AdamBrousseau ready for review, thx